code/src/languages/python/web/web_console.py

```diff
@@ -481,20 +483,45 @@ def indent(s, pad=4):
     return "\n".join(" " * pad + line for line in s.split("\n"))
 
 
-def run_doctests(f):
-    s = f.__doc__
-    if not s:
-        print(f"No doctests found for {f.__name__}, skipping...")
-        return
+def run_all_doctests():
+    export = []
+    done = set()
+    for obj in editor_ns.values():
```

Comment:
For `run_all_doctests()?` Sure, though it won't actually be runnable because web_console.py has javascript hooks that prevent it from being run in a standalone python interpreter.

Basically, we just look for anything with a `__doc__`, and assume that that corresponds to a docstring that we can parse and run. We don't actually care about the obj type, except that we want to avoid parsing the same object twice, so we use the `done` set to prevent that.